### PR TITLE
pages and router setup

### PR DESCRIPTION
**Pull request by Vishnu-Aithal**
Initial Pages And Router Set up

Preview - https://notes-app-git-pages-and-router-vishnu-aithal.vercel.app/home

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,15 +1,27 @@
...
+const root = createRoot(container);
+root.render(
+    <BrowserRouter>
+        <ThemeProvider>
+            <ConditionalRouter />
+        </ThemeProvider>
+    </BrowserRouter>
 );
+
+// ReactDOM.render(
```
Comment (hsnice16) [inline src/index.js:22]: See if these are not required, you can remove these commented code.

**Review by hsnice16 — APPROVED** [review]
UI is awesome man. Great work on that. The code is also good couldn't find much to suggest. Good job.

File: src/App.js [hunk 2]
```diff
@@ -1,5 +1,22 @@
...
+import { useTheme } from "contexts/ThemeContext";
+
 function App() {
-    return <div className="h-10 w-full bg-red-500">Test</div>;
+    const { darkTheme } = useTheme();
+    return (
+        <div
+            className={`h-screen w-screen flex flex-col transition-colors duration-300  ${
+                darkTheme ? "dark" : ""
+            }`}>
```
Comment (anik31) [inline src/App.js:12]: appreciation - good that you're using dark theme

File: src/components/Note.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+            {/* Body */}
+            <div className="my-4">Body of the Note</div>
+
+            {/* Label */}
+            <div className="flex flex-wrap gap-2 items-center my-2">
+                <Label name={"label name"} />
+            </div>
+
+            {/* Footer */}
```
Comment (anik31) [inline src/components/Note.js:36]: appreciation - good that you're providing comments for easy understanding

**Review by anik31 — APPROVED** [review]
Great work @Vishnu-Aithal . I liked the UI & the transitions, they are very nice.

File: src/index.js (hunk 1 above)
Comment (Vishnu-Aithal) [reply src/index.js:22]: Thanks for Reviewing . I will remove it .

**Pull request merged**